jenkins.sh

```diff
@@ -1,12 +1,57 @@
-#!/bin/bash -x
-set -e
+#!/bin/bash
 
-export RAILS_ENV=test
-export GOVUK_APP_DOMAIN=dev.gov.uk
+REPO_NAME=${REPO_NAME:-"alphagov/calculators"}
+CONTEXT_MESSAGE=${CONTEXT_MESSAGE:-"default"}
+GH_STATUS_GIT_COMMIT=${SCHEMA_GIT_COMMIT:-${GIT_COMMIT}}
+TEST_TASK="ci:setup:rspec default"
```

Comment:
Instead of setting `TEST_TASK` here, it would be better to set something like `DEFAULT_TEST_TASK`, and use that as a fallback below. That way it's still possible to override the task to run for running the schema tests etc. See https://github.com/alphagov/travel-advice-publisher/blob/content_ids/jenkins.sh for an example.